.github/workflows/trivy.yml

```diff
@@ -0,0 +1,70 @@
+name: Trivy scan
+
+on:
+  pull_request:
+    branches: [main, develop]
+    # Limit to certain PR event types since this action doesn't need to run for every commit
+    #types: [ready_for_review, review_requested]
+
+  # When changes are pushed to special branches
+  push:
+    branches: [main, develop]
+
+  # Allow manual triggering
+  workflow_dispatch:
+
+env:
+  GITHUB_ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+
+jobs:
+  trivy-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: "Scan configs"
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'config'
+          hide-progress: false
+          format: 'sarif'
+          output: 'trivy-config-results.sarif'
+          # ignore-unfixed: true
+          severity: 'CRITICAL,HIGH'
+      - name: Upload Trivy config scan results
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: 'trivy-config-results.sarif'
+          category: trivy-config
+
+      - name: "Set up VRO build env"
+        uses: ./.github/actions/setup-vro
+      - name: "Build jars and Docker images"
+        run: ./gradlew :app:docker -PGITHUB_ACCESS_TOKEN=${{ env.GITHUB_ACCESS_TOKEN }}
+
+      - name: "Scan rootfs"
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'rootfs'
+          format: 'sarif'
+          output: 'trivy-rootfs-results.sarif'
+          severity: 'CRITICAL,HIGH'
+      - name: Upload rootfs results to GitHub Security tab
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: 'trivy-rootfs-results.sarif'
+          category: trivy-rootfs
+
+      - name: "Scan image: app"
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'image'
+          image-ref: 'va/abd_vro-app:latest'
+          format: 'sarif'
+          output: 'trivy-image-results.sarif'
+          severity: 'CRITICAL,HIGH'
+      - name: Upload image results to GitHub Security tab
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: 'trivy-image-results.sarif'
+          category: trivy-image-app
```

Comment:
Note this is only scanning 1 of the many images we have. If this is useful, we can add the other images.